src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6924,6 +6926,17 @@ export class StudyViewPageStore {
         });
     }
 
+    @computed
+    public get displaySampleTreatments() {
+        return remoteData({
+            invoke: () => {
+                return defaultClient.getContainsSampleTreatmentDataUsingPOST({
+                    studyViewFilter: this.initialFilters,
```

Comment:
Question, isn't studyIds not enough to check they have treatments data or not? do we need studyViewFilter?